src/erasure-code/clay/ErasureCodeClay.cc

```diff
@@ -185,6 +185,19 @@ int ErasureCodeClay::decode_chunks(const set<int> &want_to_read,
   return res;
 }
 
+void ErasureCodeClay::encode_delta(const bufferptr &old_data,
+                                   const bufferptr &new_data,
+                                   bufferptr *delta)
+{
+  ceph_assert(false); // Not yet supported by this plugin
```

Comment:
Nit: I think the convention is to use ceph_abort("Not yet supported by this plugin"); instead of an assert that always fails

You could also put this implementation in ErasureCodeInterface as the default implementation and then just override it in plugins that actually support the interface. This would save having to put these stubs into all the plugins that don't support the interface.